Mu2eG4/src/Mu2eG4WorkerRunManager.cc

```diff
@@ -11,6 +11,7 @@
 
 //Framework includes
 #include "cetlib_except/exception.h"
+#include "cstddef"
```

Comment:
suggest angular brackets on cstddef include

Reply:
Honestly, I don't remember ever adding that include and I don't see why we need it now, so I am going to take it out.